dotcom-rendering/src/amp/components/Header.tsx

```diff
@@ -11,7 +11,7 @@ import {
 	visuallyHidden,
 } from '@guardian/source-foundations';
 
-import Logo from '../../static/logos/guardian-anniversary-logo.svg';
+import Logo from '../../static/logos/the-guardian-best-website.svg';
```

Comment:
Of course!

Reply:
@SiAdcock @sndrs Should this live in source-react-components or the development kitchen?

Reply:
https://github.com/guardian/source/pull/1400 PR here - perhaps we can ticket this up so that we aren't blocked waiting for this to be merged and released?

Reply:
https://trello.com/c/kNKUHQZu/342-dcr-replace-the-best-website-local-svg-with-source

Reply:
done ✅